userguide/dailyuse/location.rst

```diff
@@ -61,77 +67,101 @@ Launch the ``System Settings`` app. Select ``Security & Privacy``, then select `
+Under ``Let the device detect your location:``, select the appropriate option:
 
 * ``Using GPS`` turns on location services.
 * ``Not at all`` turns off location services.
 
 Allowing apps access to location data
 -------------------------------------
 
-Ubuntu Touch allows you to allow or deny apps access to your location data. This decision is presented the first time the app tries to access your current location. You can change your decision later.
+Ubuntu Touch allows you to allow or deny apps access to your location data.
+This decision is presented the first time the app tries to access your current location.
+You can change your decision later.
 
 An app will show the location permission request the first time it tries to access your current location.
 
 .. figure:: /_static/images/userguide/location/location-prompt.png
     :align: center
 
-Select ``Allow`` to give the app access to your current location. Select ``Don't Allow`` to deny the app access to your current location. Some app features may not work correctly if you select ``Don't Allow``.
+Select ``Allow`` to give the app access to your current location.
+Select ``Don't Allow`` to deny the app access to your current location.
+Some app features may not work correctly if you select ``Don't Allow``.
 
 Changing your permissions
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can change your decision to allow an app access to your location data. This is useful when you gave an app access to your location but you'd like to deny access, or if you denied access but would like to give access.
+You can change your decision to allow an app access to your location data.
+This is useful if you have granted an app access to your location but would now like to revoke it, or if you denied access but would like to grant access again.
 
 There are several ways to find these controls.
 
 Location settings
 """""""""""""""""
 
-`Open location settings in the settings app`_. All of the apps which have requested access to your location are listed under ``Let apps access this location:``. Toggle the switch next to an app "on" to allow it to access your location data. Toggle the switch "off" to deny access.
+`Open location settings in the settings app`_.
+All apps that have requested access to your location are listed under ``Let apps access this location:``.
+Toggle the switch next to an app "on" to allow it to access your location data.
+Toggle the switch "off" to deny access.
 
 App permission settings
 """""""""""""""""""""""
 
-Launch the ``System Settings`` app. Select ``Security & Privacy``, then select ``App permissions``, followed by ``Location``. All of the apps which have requested access to your location are listed. Toggle the switch next to an app "on" to allow it to access your location data. Toggle the switch "off" to deny access.
+Launch the ``System Settings`` app.
+Select ``Security & Privacy``, then select ``App permissions``, followed by ``Location``.
```

Comment:
This isn't a correct use of ampersand, and I think it changed in the app strings.
Will have to check.